src/ReactNative/Components/Image.purs

```diff
@@ -21,19 +22,23 @@ type ImageProps r = {
 
 type ImagePropsO = BaseProps (
     style :: Styles
+  , blurRadius :: Number
+  , loadingIndicatorSource :: Number
```

Comment:
array of ImageSourcePropTypes, number
Can accept a number as returned by require('./image.jpg')

add this as comment